hack/update-codegen.sh

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -x
```

Comment:
Spurious change

Reply:
Ah, indeed. Though it's pretty hard to debug docs generation without it.